ci/dra_aarch64.sh

```diff
@@ -45,22 +41,25 @@ case "$WORKFLOW_TYPE" in
             # e.g: 8.0.0-alpha1
             STACK_VERSION="${STACK_VERSION}-${VERSION_QUALIFIER_OPT}"
         fi
-	;;
+        info "Build complete, setting STACK_VERSION to $STACK_VERSION."
+        ;;
     *)
-        exit 1
-	;;
+        error "Workflow (WORKFLOW_TYPE variable) is not set, exiting..."
```

Comment:
yes, the error function handles the exit 1.

Reply:
I don't know the `bash` best practices, but I would communicate in someway that `error` also exit the execution.
We could extract the `exit 1` or rename it in `error_and_exit`

Reply:
I'll consider that for the next iteration of fixes, because as it stands the script will abort operation in case of an error. Also, there's only one place where it is being used.